lib/msal-angular/package.json

```diff
@@ -33,23 +33,21 @@
   "main": "./dist/bundles/azure-msal-angular.umd.js",
   "typings": "./dist/azure-msal-angular.d.ts",
   "beachball": {
-    "disallowedChangeTypes": [
-      "major"
```

Comment:
Let's restore the disallowedChangeTypes everywhere as well

Reply:
Actually this can wait until we release, but let's make sure this is tracked, if not already, so we don't forget

Reply:
Yes, it's tracked on ADO.